firebase-firestore/src/main/java/com/google/firebase/firestore/local/OverlayedDocument.java

```diff
@@ -22,21 +22,27 @@
 public class OverlayedDocument {
   private Document overlayedDocument;
   private FieldMask mutatedFields;
```

Comment:
should we make this `@Nullable private FieldMask mutatedFields;` ?

Reply:
Done.